api/v1alpha1/release_types.go

```diff
@@ -38,11 +39,36 @@ type ReleaseSpec struct {
 	ReleasePlan string `json:"releasePlan"`
 }
 
+// ReleaseReason represents a reason for the release "Succeeded" condition.
+type ReleaseReason string
```

Comment:
I think this ReleaseReason stuff was a rebase error. I don't see it in this file in main